examples/bakdata/atm-fraud-detection/defaults.yaml

```diff
@@ -25,8 +34,8 @@ streams-app:
   app:
     labels:
       pipeline: ${pipeline_name}
-    streams:
-      optimizeLeaveGroupBehavior: false
+    command_line:
```

Comment:
```suggestion
    commandLine:
```

Reply:
both works :)

Reply:
I know :) I was referring to the conversation we had. Especially for the examples and docs, we should stick to only one naming convention